package.json

```diff
@@ -27,7 +27,6 @@
   "devDependencies": {
     "@types/jest": "^23.3.9",
     "@types/node": "^10.12.5",
-    "@types/tone": "https://github.com/Tonejs/TypeScript.git",
```

Comment:
You mentioned TypeScript definitions come from TypeScript branch. I don't see you select the branch anywhere here? Can you explain?

Reply:
I changed Tone version to `14.5.23`. 

Here https://www.npmjs.com/package/tone this version is tagged as "typescript" (under "512 versions" tab)

Reply:
Ah, that's how it works. Thank you for showing me this.